realm/realm-library/src/objectServer/java/io/realm/PermissionManager.java

```diff
@@ -518,6 +560,82 @@ public void cancel() {
         }
     }
 
+    private class MakeOfferAsyncTask extends AsyncTask<String> {
```

Comment:
I would *SOOOOO* love it if this thing were renamed `RealmAsyncTask`.  An `AsyncTask` is a central Android thing.  I hate it that we are overloading the name.